### DISCUSSION TITLE
دعوت از نخبگان و جامعه علمی برای تشکیل جبهه مردمی

### DISCUSSION THREAD

**Discussion by allahyar** [WeAreMahsaAmini/Core#156] [Ideas]
درود بیکران،
من بابت واقع بین بودنم و این صبحت ابتدا عذرخواهی میکنم ،تجربه حضور در اعتراضات 88 و 98 و این روزها برای مهسا امینی را دارم من بدنبال این بود بودم که چرا پس از 88 دیگر حضور میلیونی رقم نمیخورد و مشارکت خیابانی بنسبت دیگر سال ها کمتر بود با اینکه مطالبات فراتر از رای مرا پس بده بود  تحقیق میدانی کردم از اکثر نسل ها این پرسش را کردم در دهه 70 و 60  ، علت را عدم وجود لیدر برای ترس از سرکوب و مهمتر آلترناتیو بود عموما میگفتن فراخوان ها در یک نقطه است طبیعی است که سرکوب گران زودتر از ما حاضر و با استراتژی هستند یک نفر در این اعتراضات نبود بگوید مردم فقط کافیه بیاید در خیابان خود لازم نیست به نقطه که تعیین میشود بیایید. همه میگفتن ما هم دوست داریم این رژیم سقوط کند اما نمیخواهیم جای پدرهای قدم بگذاریم .
این را میدانم به نسبت سایر جنبش و اعتراضات ما از حمایت دنیا برخوردار بودیم جامعه علمی ، فرهنگی و هنری ، ایرانیان مقیم خارج و دانشجویان و ...  هم سهم بزرگی ایفا کردن اما بنظر من این کافی نیست.
واقعیت این است که مردم بر خلاف رژیم باید شایسته سالار باشن و این مستلزم شناخت کارشناسان و متخصصان است. من این ایده به ذهنم خطور کرد که یک بستری همانند انجمن اجرا بشه به هدف شناسایی مشکلات و رفع آن ها بعنوان مثال یه تاپیکی در دسته بندی بحران های زیست محیطی ایجاد بشه به اسم مثلا بحران آب ورشکستگی آبی از متخصصان و کارشناسان دعوت شود که گفتگو کنند و بدنبال راهکار باشند. مردم هم زیر هر راهکار تعامل داشته باشن همانند همین دیسکاشنز و قابلیت رای گیری یا مثلا در مورد ماده n قانون اساسی یا اصلا بازنویسی آن یا در مورد نظام سیاسی آینده با مشارکت جامعه شناسان . این چیزهایی که مردم بصورت منسجم نیازمند به آگاهی آنن شایستگی را اینجا میتوان یافت.

ما نیاز به آگاهی ، اتحاد و حضور میلیونی داریم .

پاینده باد ایران

**Comment by AnonymousMaximus2** [comment]
این چیزی که شما درباره اش نوشته اید حالت ایده آل و مدرن دولت الکترونیک است

و دولت الکترونیک باید زیربنای فنی، اقتصادی، مدیریتی اش قوی باشد.

با جبهه مردمی و این چیزها کار به جائی نمیرسد، باید یک تشکیلات بین المللی تاسیس شده و بودجه کافی داشته باشند.

روسیه و چین پشت دولت ایران هستند و از طرفی صدها میلیارد دلار درآمد نفتی توی جیب دولت و حکومت ایران است که با این پول ارتشهای دنیا را میتواند بخرد.

با دست خالی و با تعداد مردم کم نمیشود. 

زمان انقلاب 1357 ارتش کنار کشید، شاه از ایران رفت و مردم جسور تر ا امروز بودند و طرفداران حکومت سکوت کردند.

الان طرفداران حکومت تعدادشان خیلی زیاد است، سپاه در لباس ارتش است، و ....

یک فکر بهتری بکنید. 

برای یک فکر بهتر اولین قدم یک سازمان بین المللی مطالبه گر و پیگیر با بودجه کافی است.

**Reply by allahyar** [reply]
احساس میکنم نتوانستم لپ مطلب را انتقال دهم ببینید این چیزی که من نوشتم برای بعد از گذر از جمهوری اسلامی نیست من نمیدانم شما ایران هستید یا خیر اما مردم در تاریکی مطلق به سر میبرند عده ای یک شمع به دست گرفته اند و مسیر را تنها براندازی میدانن و پلنی برای بعد وجود ندارد و عده ای زیادی از سر اینکه هیچ لیدر و الترناتیوی ندارن به کف خیابان نمی آیند گفتم اعتراضات نباید متمرکز در چند نقطه باشد و قتی اعتراضات غیر متمرکز باشد مدیریت سرکوب سخت تر است حال شما میگوید با نیروهای نیباتی امثال حشد الشعبی یا حزب ا... یا اصلا مداخله روسیه و چین ما باز هم سرکوب میشویم پاسخ من خیر است چرا چون 88 و 98 کسی مداخله نکرد اما میپذیرم که از نیروی نیابتی کمک گرفتن.

کلا انقلاب در برخی کشورها مثل سوریه که از لحاظ ژئوپولتیک مهم هستن به ثمر نمیشنند ابر قدرت ها برای منافعشون مداخله نظامی می کنند این منطقی تر است تا صحبت شما سوریه نه تنها برای جمهوری اسلامی مهم بود بلکه برای شاه هم حائز اهمیت بود و حمایت مالی 300 میلیون دلاری داشته همان حافظ اسد دست به قتل عام  10 هزار سوری معترض زد.

**Comment by kaveh6** [comment]
دوست عزیز ، الان شرایط همینی هست که میبینین ، مردم هرچی داشتن All-In دارن میکنن . طرفداران حکومت هم تعدادشان خیلی زیاد نیست . آدمهای منفعل تعدادشان خیلی زیاد است . شما اگه حرفت اینه ، به نظر من بهتره سکوت کنی و ترک کنی جمع رو . الان وقت زدن این حرفا نیست ، به درد کسی هم نمیخوره شنیدن این حرفا توی این شرایط . اون موقع فلان بود ، الان ارتش فلانه . اصلاً هیچ کدوم از این حرفا الان مهم نیست . یه چیز اهمیت داره و اونم اینه که هر کی هر جوری میتونه با هر منابعی که دم دستش داره یه گوشه کار رو بگیره . 
اگه نقد سازنده ای داری در مورد ایده بفرمایین ، اگرم نه که مجدد میگم سکوت بهتره تا زدن حرفای اینچنینی . 

@Investmentbanker

**Reply by AnonymousMaximus2** [reply]
توصیه میکنم کتاب هنر جنگ نوشته سان تزو را بخوانید،

2500 سال پیش نوشته ولی الان خیلیها قوانین جنگیدن را بلد نیستند و برای مبارزه با حکومت ایران با چوب و چماق وارد میدان میشوند.

من حتی به شما هم اعتماد ندارم،

انقدر چیزها دیده ام که مطمئنم تا حساب شده حرکت نکنید مثل دفعه قبل همه را درو خواهند کرد.

شما از قراردادهای بین المللی ایران و پیمانهایی که با چین و روسیه بسته چه میدانید؟

وقتی میگوئید سکوت کنم، یعنی اگر کسی چیزی بلد باشد و حرف نزند آب به آسیاب دشمن ریختن است.

توی چنین مواقعی وقتی شیر تو شیر میشه اکثرا نفوذی های دشمن وارد جمع انقلابی ها میشوند و از درون تخریب و تضعیف میکنند.

من میگم تشکیلات مدیریتی و سازماندهی شده درست کنید که جای خالی یک رهبر را بتونه بگیره

شما میگید سکوت کنم؟

مردم ایران نیاز به یک رهبر جسور دارند مانند گاندی، مانند نلسون ماندلا، و ...

با کنار ایستادن و حرف زدن کار پیش نمیره

و با اینکه هر کسی هر کاری بتواند انجام بدهد هیچ عقل سالمی قبول نمیکند،

کتاب هنر جنگ 2500 سال پیش نوشته شده ولی در تمامی دانشگاههای دنیا در اروپا، آمریکا و چین و روسیه تدریس میشه،

حرفهای شما بر خلاف و ضد مهمترین اثر علمی جنگ است.

اگر با سکوت شما پیروز میشوید من سکوت میکنم.

**Reply by Moonaramoona** [reply]
جدای بحث رهبر، ما نیاز به دریافت اطلاعات درست داریم که یاد بگیریم. بحث شناسایی گروه نخبه هست که همیشه در خفا هستن و صداهاشون نمیشنویم

**Reply by kaveh6** [reply]
من مخالف شما نیستم . مخالف اینم که آدما فقط حرف بزنن ، باید عمل باشه . 
الان اگه بخوایم به این فکر کنیم که اگه رهبر نباشه ، نمیشه . پس خوب نمیشه . چون هیچ رهبری وجود نداره که همه ی مردم قبولش داشته باشن و فالوش کنن . 
به نظرم تغریفمون رو از رهبر باید روشن کنیم ، به نظر من رهبر نمیتونه یه شخص باشه ، باید یه ائتلاف باشه ، باید یه گروه باشه . 
رهبر کسی نیست که مردم چشم و گوش بسته تکریم کنن و دنباله رو باشن ، توی این مقطع ما نیاز داریم که گروهی جهت دهی کنه این movement رو . 
ماها ذاتاً دنبال بت سازی از اطراف بودیم توی تمام تاریخمون ، منتظر بودیم که یکی بیاد مارو نجات بده . این توی ما ریشه داره . 
نظر شخصی من اینه که اگه کسی راه حلی داره ، بیاد وسط حرف راه حل بگه . این به درد کسی نمیخوره که این کاری که داری میکنی به درد نمیخوره ، باید فلان اتفاق بیفته . اگه این حرف رو بخوای بزنی همچنان میگم بهتره سکوت کنی چون کسی که بخواد action داشته باشه دوست نداره این حرف رو بشنوه . 
اگه کسی میخواد دعوت کنه از نخبگان ، بکنه اگه امکانش رو داره . یا اگه کسی فکر میکنه که میتونه کمک کنه توی این پروژه ، بیاد کمک کنه . 
ولی این که بیای بگی نه چون فلان ... کمک که نکردی هیچ ، داری سنگ اندازی میکنی .

**Reply by kaveh6** [reply]
مهم نیست به من اعتقاد داشته باشی یا نه ، تو منو نمیشناسی منم تورو نمیشناسم .

**Comment by kaveh6** [comment]
اتفاقا ایده به نظر من ایده ی خوبی هم هست . حداقل حسنی که داره که اینه که عادمهای متخصص دور هم جمع میشن و همدیگه رو میشناسن و awareness بیشتری ایجاد میکنن . 4 نفر هم مثل من که خیلی تخصص ندارن غیر از حوزه ای که توش کار میکنن ایده ها رو میشنون و آدم های درست حسابی رو میشناسن .

**Reply by allahyar** [reply]
ممنونم صحبت های شما کاملا درسته، همین بستری که ما در حال تعامل هستیم اگر عملگرا و سازنده باشه قطعا اون بستری که ما برای اونها ایجاد میکنیم هم همین تاثیر رو داره و اون تاثیر گذاری رو افرادی مثل سیاوش، کوشیار و ... ایجاد کردن و بماند که اگر رسانه ای بشه بازتابش چه لرزه ای به اندام رژیم می اندازه  دیگه سر همین محیط زیست ببیند براش آدم حذف میکردند

**Comment by Moonaramoona** [comment]
بنظر من هم ایده خیلی خوبیه. من همیشه به این موضوع فک کردم و میکنم. مشکل ما این هست که قشر نخبه، باتجریه وبا سواد رو در فیلدهای مختلف نمیشناسیم و صداهاشون نمیشنویم . خیلی خوب مبشه که یه دایرکتوری وجود داشته باشه بر اساس موضوع و حوزه های مختلف تقسیم بندی بشه و افراد الیت اون حوزه که رزومه سفیدی هم دارن بهشون اضافه بشه و یک خلاصه ای از زندگی و فعالیتهاشون هم گفته بشه و شبکه های اجتماعیشون هم باشه که ما بهشون کانکت بشیم. مثلا گروهها میتونه اقتصاد، تکنولوژی، سلامت و درمان، سیاست، محیط زیست، سیاسی و اجتماعی و حیلی چیزهای دیگه باشه. ما به اتاقهای فکر نیاز داریم. همینکه بشناسیمشون و فالوشون کنیم قدم بزرگیه. بنظرم ما در قدم اوی به آرشیو اسمها و آشنایی نیاز داریم. هدف این باشه که بشناسیمشون. یکی مثل کوشیار که در حوزه تک شناختیم و کلی چیز تا الان یاد گرفتیم ازش. در حوزه های دیگه هم نیاز داریم بشناسیم و اونا هم همو بشناسن

**Reply by AnonymousMaximus2** [reply]
آدمهای نخبه اکثرا نمیخوان شناخته بشن چون آسیب پذیر تر هستند و طرفداران حکومت آنها را به راحتی میزنند

افراد عادی که آسیب پذیری کمتری دارند حتی براشون وسط خیابان شعار دادن هم اهمیتی نداره 

ولی کسی که میتونه میلیونها نفر ادم را هدایت کنه، به فکر خودش نیست بلکه مراقب حفاظت از استعدادها و توانمندیهای خودش است که بتواند سر فرصتی ازش استفاده کند.

افرادی که اکثرا می آیند و سعی میکنند مردم را دور خودشان جمع کنند ، نخبه ها و افراد متخصص واقعی نیستند.

یک نسخه فرد نخبه میتواند میلیونها انسان را مسیرشان را هدایت کند، ولی اولا نمیتواند ادعا کند که نخبه است و با دیگران فرق دارد

دوما اگر هم ادعایی بکند افراد حکومتی او را تخریب میکنند و مردم که نمیتوانند تشخیص بدهند، به راحتی او را ترک میکنند.

و به همین راحتی ترور شخصیتی میشود.

افراد نخبه با باتوم و گلوله کشته نمیشوند، آنها با یک شایعه و تخریب شخصیتی ترور شده و تا آخر عمرشان در جامعه منزوی و ناشناخته باقی میمانند. و اصلا نیاز به دستگیری توسط حکومت ندارند.

مردم هستند که باید انقدر دانش و سواد داشته باشند که در مقابل ترفندهای جنگ روانی بتوانند نخبه را از نفوذی تشخیص بدهند و دنبال نخبه بروند.

این موضوع جدیدی نیست و هزاران سال است بشر این موضوع را نتوانسته حل کند که به سطحی از آگاهی و درک و شعور برسد که بتواند فرق بین نفوذی را با نخبه واقعی خیرخواه تشخیص بدهد.

**Reply by kaveh6** [reply]
خوب الان بگو به من این حرفایی که الان زدی که چی ؟ نتیجه گیری کن از اینا . یه سری داستان گفتی ... نه راه حل دادی ، نه نتیجه گیری کردی ، فقط غر زدی .

**Reply by kaveh6** [reply]
من فکر میکنم شما هم جزو کسانی هستی که جمهوری اسلامی رو نمیخوان و شهری و مافیا نیستی . ولی بهتره که شهروند تبر نباشیم . اگه بلدیم یا تخصصی داریم ، حرفای به درد بخور بزنیم . کل حرف من همینه .

**Comment by AnonymousMaximus2** [comment]
یک مروری اگر به انقلاب 57 بیاندازید متوجه میشوید که انقلاب 57 با راهپیمائی و شعار مردم پیروز نشد

بلکه با شایعات و ترور شخصیتی که خمینی و افراد وابسته به خمینی در قالب کمپین چند ساله راه انداختند باعث ایجاد نفرت بین مردم شدند و صدها هزار نفر را که فکر میکردند شاه میخورد و آنها گرسنه هستند به خیابان ریختند و میلیونها نفر هم سکوت کردند و حرفی نزدند.

همیشه در تمامی انقلابها 90 درصد پیروزی از طریق ترور شخصیتی و جنگ روانی و رسانه ای انجام میشود

و فقط 10 درصد فعالیت نظامی و گلوله است.


پس اگر میخواهید پیروز شوید و جبهه تشکیل بدهید اول چاره ای به فکر جنگ روانی و ترورهای شخصیتی بکنید

ببینید چه عاملی باعث شد تا شاه ایران در برابر آن همه خدماتی که کرد، ناسپاسی و قدر نشناسی مردم و شعارهای ناهنجار مردم باعث شد دلش شکست و از ایران رفت. 

الان هم دوباره فردی لایق تر از شاه بیاید اگر مردم آگاهی کافی نداشته باشند و نتوانند فرق بین دلسور را با اختلاسگر تشخیص ندهند هیچ اتفاقی نخواهد افتاد.

در بین تمامی شورشی ها چه در اینترنت و چه در خیابانها عده ای نفوذی هستند که دنبال خبر جمع کردن هستند.

حتی اینجا نیز عده ای هستند که به نام ضد حکومتی آمده اند و دنبال جمع آوری اطلاعات و خنثی کردن فعالیت مردم هستند.

**Reply by ghost** [reply]
محمدرضا دلش شکست؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟
داداش فرار کردن میدونی یعنی چی؟😂
تعریفت از فرار دقیقا چیه؟

**Reply by ghost** [reply]
البته عادت کردین به سانسور کردن حقایق!

**Reply by allahyar** [reply]
برداشت من از شما ایته ظاهرا سلطنت طلب هستید درسته ؟

**Reply by AnonymousMaximus2** [reply]
> برداشت من از شما ایته ظاهرا سلطنت طلب هستید درسته ؟

مشدی از خواب بیدار شو،

توی تفکرات انقلابی 50 سال پیش داری غلط میزنی

الان سال 2022 هست.

انقلاب الان ایران انقلاب مدرن هست،

الان رسانه های شبکه های اجتماعی، هوش مصنوعی، شبکه ماهواره ای استار لینک، 
مطالعات علمی و مهندسی گسترده در مورد نحوه انجام انقلاب جهت افزایش احتمال پیروزی و حد اقل رسانی

این بار مهندسی انقلاب انجام شده و کلی محاسبات انجام شده، 

به قول پسر شاه که چند روز پیش در سخنانش گفت این تو بمیری با اون تو بمیری فرق داره

دیگه زمان سلطنت طلب ها و فرقه گرائی ها به سر اومده 

این انقلاب بدون رهبر و بدون گرایش و بدون طلبیدن هایی مانند سلطنت طلبهاست.

همه دنیا با هم متحد شده اند، فرقه گرائی و تفکیک جنسیتی و تفکیک گروه های سیاسی مال احمق هاست که ضریب هوشی پائینی دارند و میخواهند هر کسی را به فرقه ای و یا گروهی نسبت بدهند تا بتوانند در مغز ناقص خود همه را دسته بندی و کلاسه بندی کنند و تجزیه تحلیل کنند و بفهمند کی به کیه

در حالی که این بار با بارهای قبلی فرق داره

این بار خیلی از مذهبی ها و حتی روحانیان قم و مشهد هم جزء انقلابی ها هستند.

**Reply by AnonymousMaximus2** [reply]
> محمدرضا دلش شکست؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟؟ داداش فرار کردن میدونی یعنی چی؟😂 تعریفت از فرار دقیقا چیه؟

اجداد نحس شما ناشکری کردند، و کفنران نعمت کردند، آن قدر مرگ بر شاه گفتند تا چاره ای جز رفتن نداشت،
و امثال شما رفتنش را نامش را گذاشتید فرار کرد.

اگر اسم این کار فرار کردن است، پس پیامبر اسلام هم از مکه به مدینه فرار کرد؟

بارها از فیلم ضبط شده شاه شنیدم که گفت اگر انقلاب شود مردم ایران به 500 سال عقب بر میگردند.

الان اگر دروغها را کنار بگذاریم، ایران اگر انقلاب نمیشد الان از همه دنیا جلو بود. ولی از همه دنیا عقب است.
گرچه در ظاهر به دروغ که عادتش است ادعا میکند از همه جلو است.

**Reply by ghost** [reply]
نکته اول: شاید بظاهر در مقایسه اقتصاد ایران در قبل قبل از انقلاب و بعد از انقلاب، دوران شاه سنگینی کند اما باید توجه کرد که زمان شاه برای ایران دقیقا عصر استعمار، غارت و بندگی در برابر انگلیس بود که ایران درحال تبدیل شدن به یک عروسک خیمه شب بازی برای غرب میشد.
نکته دوم: قیاس اقتصاد الان و زمان شاه بدون در نظر گرفتن همه نکات غیرممکن است. دستاوردهای امروزه ایران باتوجه به این همه تحریم از طرف غرب گرانبهاست. معلومه که وقتی نفت و گاز و خلیج فارسمون رو بدیم به کشورهای استعمار گر اقتصادمون هم آزاد میشه و در زمان رضا شاه قطعا نمیشد آینده خوبی رو برای ایران متصور بود.
نکته سوم: طغیان رضا شاه: فسق و فجور آشکارای شخص اول مملکت، کشیدن حجاب از سر زنان مملکت، از بین رفتن تدریجی فرهنگ اصیل ایرانی و جایگزین شدن فرهنگ غربی

**Comment by ghost** [comment]
#no_to_sedition
#نه_به_آشوب
#نه_به_اغتشاش
#صلح
#مرگ_بر_آشوبگر

**Reply by AnonymousMaximus2** [reply]
تکبیییر :) 


اون مهدی 313 بزنه توی سرت که توی 1400 سال از زمان اسلام تا الان 313 نفر آدم پیدا نکردی تا امام زمانت ظهور کنه و امثال تو را از منجلاب طاغوت جمهوری اسلامی نجات بده

**Comment by AnonymousMaximus2** [comment]
> درود بیکران، من بابت واقع بین بودنم و این صبحت ابتدا عذرخواهی میکنم ،تجربه حضور در اعتراضات 88 و 98 و این روزها برای مهسا امینی را دارم من بدنبال این بود بودم که چرا پس از 88 دیگر حضور میلیونی رقم نمیخورد و مشارکت خیابانی بنسبت دیگر سال ها کمتر بود با اینکه مطالبات فراتر از رای مرا پس بده بود تحقیق میدانی کردم از اکثر نسل ها این پرسش را کردم در دهه 70 و 60 ، علت را عدم وجود لیدر برای ترس از سرکوب و مهمتر آلترناتیو بود عموما میگفتن فراخوان ها در یک نقطه است طبیعی است که سرکوب گران زودتر از ما حاضر و با استراتژی هستند یک نفر در این اعتراضات نبود بگوید مردم فقط کافیه بیاید در خیابان خود لازم نیست به نقطه که تعیین میشود بیایید. همه میگفتن ما هم دوست داریم این رژیم سقوط کند اما نمیخواهیم جای پدرهای قدم بگذاریم . این را میدانم به نسبت سایر جنبش و اعتراضات ما از حمایت دنیا برخوردار بودیم جامعه علمی ، فرهنگی و هنری ، ایرانیان مقیم خارج و دانشجویان و ... هم سهم بزرگی ایفا کردن اما بنظر من این کافی نیست. واقعیت این است که مردم بر خلاف رژیم باید شایسته سالار باشن و این مستلزم شناخت کارشناسان و متخصصان است. من این ایده به ذهنم خطور کرد که یک بستری همانند انجمن اجرا بشه به هدف شناسایی مشکلات و رفع آن ها بعنوان مثال یه تاپیکی در دسته بندی بحران های زیست محیطی ایجاد بشه به اسم مثلا بحران آب ورشکستگی آبی از متخصصان و کارشناسان دعوت شود که گفتگو کنند و بدنبال راهکار باشند. مردم هم زیر هر راهکار تعامل داشته باشن همانند همین دیسکاشنز و قابلیت رای گیری یا مثلا در مورد ماده n قانون اساسی یا اصلا بازنویسی آن یا در مورد نظام سیاسی آینده با مشارکت جامعه شناسان . این چیزهایی که مردم بصورت منسجم نیازمند به آگاهی آنن شایستگی را اینجا میتوان یافت.
> 
> ما نیاز به آگاهی ، اتحاد و حضور میلیونی داریم .
> 
> پاینده باد ایران

انقلاب فعلی ایران انقلاب مدرن نامیده شده است زیرا که چیزهایی در آن وجود دارد که در انقلابهای قبلی جهان وجود نداشت

- عدم وجود رهبر در انقلاب
- 
- عدم وجود جبهه و حزب خاص سیاسی
- 
-  پراکنده بودن اعتراضات در مقابل راهپیمائی های میلیونی قدیمی 

- من نمیدانم پشت پرده چه کسی این برنامه ریزی ها را کرده، ولی هر که بوده بسیار باهوش و نابغه و نخبه بوده یا بوده اند.

در مقاله ای که توسط دانشگاههای بین المللی مطالعه و منتشر شده بود خواندم که اکثریت انقلابهای جهان که منجر به تولید دیکتاتوری شده اند در انقلاب خود رهبر داشته اند ولی در انقلابهای موفق که پس از پیروزی مردم به رفاه و ارامش رسیده اند، رهبری واحد و متمرکزی نداشته اند.

همچنین طبق اصول جنگ ، پراکنده بودن مردم باعث خسته شدن و استهلاک نیروهای ضد شورشی شده و آنها را از پای در می آورد.

و متمرکز بودن مردم خط تیرباران و کشتار انبوه مانند کشتار میدان شهدای تهران را خواهد داشت.

**Reply by vovak1401** [reply]
خوب این از روز اول با شعار اصلی زن زندگی آزادی شروع شد
که صلح آمیز، فرا مرزی، فرا قومی و ملیتی است 
احساسات هر انسانی رو جریه دار می کنه .

ولی با این احتیاج به رهبری مستقیم نداره فقط احتیاج به هدایت داره تا به سمت استهلاک و خسته نیرو های انقلابی نره و بعد از پیروزی مسیر برای ایجاد دمکراسی هموار باشه و نیروهای سرکوب خسته و منفک بشن
مبارزه مدنی از سوی قشر خاکستری و صد معبر و مختل کردن رفت و آمد ، ایجاد اختلاط در شبکه سوخت و انرژی می تونه آسیب شدید و اختلال اعزام نیرو بشود و کمترین تلفات برای مبارزین

این راهکار ها از سوی افراد مجرب و مقبول تحت عنوان شورای گذار تشکیل شه.

**Comment by Zohreh-sudo** [comment]
سلام 
من هم خیلی علاقمند هستم به این موضوع و دوست دارم همکاری کنم. میشه لطفا توضیح بدید که چطوری می تونم همکاری کنم؟ مرسی

**Reply by allahyar** [reply]
درود دوست عزیز این پروژه توسط سیاوش استارت زده شده و از آدرس 
future.wearemahsaamini.com
در دسترس است
🌹

**Comment by vovak1401** [comment]
من چند سال پیش شنیدم ، ترامپ فراخوان داده بود از نوابغ و رجال ایرانی و مقیم خارج ایران جمع شوند و تشکیل ائتلاف بدن تحت عنوان گروه "ققنوس" ولی دیگه نمی دونم چی شد الان هم شما هم خودتان تحقیق کنید اگه صحت داشته باشه دوباره می شه فراخوان داد ، همون جوری یکی از دوستان اشاره کردن جنگ روانی مهم نیتریت و موثر ترین روش است 
ولی ایراد گرفته شده بر دولت جدید و ائتلاف و یا هر چی اسمش رو بزارید روحیه خیلی خوب برای مبازران و تضعیف روحیه سرکوبگران است و حتی می تونه اتحاد بیشتر برای خارج نشینان و آپوزاسیون بشه و در حالت ایده آل می تونه تکیه گاهی برای ارتش ایران و همون ۱۰% که بیان شده باشه
